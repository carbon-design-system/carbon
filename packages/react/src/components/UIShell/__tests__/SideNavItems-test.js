/**
 * Copyright IBM Corp. 2016, 2018
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 */

import React from 'react';
import { mount } from 'enzyme';
import { SideNavItems, SideNavItem } from '../';

describe('SideNavItems', () => {
  let mockProps;

  beforeEach(() => {
    mockProps = {
      className: 'custom-classname',
      children: <SideNavItem>test</SideNavItem>,
    };
  });

  it('should render', () => {
    const wrapper = mount(<SideNavItems {...mockProps} />);
    expect(wrapper).toMatchSnapshot();
  });
});
