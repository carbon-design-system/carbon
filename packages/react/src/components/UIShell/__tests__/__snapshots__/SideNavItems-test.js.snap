// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SideNavItems should render 1`] = `
<SideNavItems
  className="custom-classname"
>
  <ul
    className="bx--side-nav__items custom-classname"
  >
    <SideNavItem
      key=".0"
    >
      <li
        className="bx--side-nav__item"
      >
        test
      </li>
    </SideNavItem>
  </ul>
</SideNavItems>
`;
